towncrier.toml

```diff
@@ -1,9 +1,10 @@
 [tool.towncrier]
+    version = "1.99.4"
```

Comment:
In Element, version was not declared here, do you know why ? Maybe it is only set locally when running towncrier and not pushed to the repository ?

Reply:
i think they add it each time they create a new version.